language-design.md

```diff
@@ -17,7 +17,11 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+In “Growing a Language” by Guy Steele, the author argues: “In a race, a small language with warts will beat a well designed language because users will not wait for the right thing; they will use the language that is quick and cheap, and put up with the warts. Once a small language fills a niche it is hard to take its place.” [Steele, 1998]. We found this interesting and compelling. When we dream about creating our own programming language, we want everything to be perfect. We are tempted to spend countless hours pouring over our code trying to make it perfect, scoffing at “bad programmers” who have create ugly, error-prone and unintuitive languages or programs. But when we consider this quote, it’s difficult to deny its truth. Creating a perfect programming language is impossible, and even making a very good language is in some ways doomed to failure. No one will wait for you to build the perfect system. Even if you succeed, by the time you do, people will have adopted a different language, become accustomed to its quirks, and found ways to work around them. They will not want to invest the time and energy in learning to use a “better” programming language if the language thy already have works well enough for their purposes.
 
```

Comment:
Remember this, when it comes time for the projects :). Also, a talk called [_The Birth and Death of JavaScript_](https://www.destroyallsoftware.com/talks/the-birth-and-death-of-javascript) discusses this same idea for that language.